dbt_subprojects/daily_spellbook/models/near/v1_signer/near_v1_signer_users.sql

```diff
@@ -0,0 +1,36 @@
+{{
+  config(
+    schema = 'near_v1_signer',
+    alias = 'users',
+    materialized = 'view',
+    post_hook = '{{ expose_spells(\'["near"]\',
+                                    "project",
+                                    "v1_signer",
+                                    \'["bh2smith"]\') }}'
+  )
+}}
+
+-- PoC Query: https://dune.com/queries/4642523
+SELECT
+  receipt_predecessor_account_id as sign_for,
+  json_extract_scalar(action_function_call_args_parsed, '$.request.path') AS derivation_path,
+  CAST(json_extract_scalar(action_function_call_args_parsed, '$.request.key_version') AS INTEGER) AS key_version
+FROM {{ source('near', 'actions') }} action
+JOIN {{ source('near', 'logs') }} log
+ON action.block_height = log.block_height
+AND action.receipt_id = log.receipt_id
+WHERE
+  receipt_receiver_account_id = 'v1.signer'
+  -- Deployment block of v1.signer: https://nearblocks.io/txns/79BBi3H3XgzktscGqKDZHAiNGRseuAJuYCmHkxKFLNif
+  AND action.block_height >= {{BlockHeight}}
+  AND log.block_height >= {{BlockHeight}}
```

Comment:
{{BlockHeight}} isn't actually assigned anywhere in this model, so this will break.

i would rather introduce a CTE above final query which grabs the required `block_height` value and inner join here instead. or, if it's a hardcoded date, feel free to introduce variable for start date and apply variables here (easier if never changing). example [here](https://github.com/duneanalytics/spellbook/blob/90c8baf69e7dc083ec5c21a8b7135c61b46f0497/dbt_subprojects/solana/models/_sector/dex/dex_solana_price_hour.sql#L18).

Reply:
Sorry that was an old query parameter that got pasted in here. Will put the real one.

Reply:
[7a127fc](https://github.com/duneanalytics/spellbook/pull/7597/commits/7a127fcbd03d8c3bd4c2dd38ed48bea3c27f4894)